pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -170,6 +178,36 @@ func getClients() (*vCenterClient, error) {
 	}, nil
 }
 
+// checkComputeResource checks the ClusterComputeResource to see if there are any attached hosts.  Without
+// hosts, the check for networks will fail.  We want the message to be clear if this is the cause so
+// users know what to address.  This only returns an error if unable to verify or if no ComputeResources
+// are found.
+func checkComputeResource(ctx context.Context, datacenter string, cluster string, finder Finder) error {
+	ctx, cancel := context.WithTimeout(ctx, 60*time.Second)
+	defer cancel()
+
+	path := fmt.Sprintf("/%s/host/%s", datacenter, cluster)
+	ccr, err := finder.ClusterComputeResource(context.TODO(), path)
+	if err != nil {
+		return err
+	}
+
+	var ccrMo mo.ComputeResource
+	logrus.Debugf("CCR: %v", ccr)
+
+	err = ccr.Properties(ctx, ccr.Reference(), []string{"host"}, &ccrMo)
+	logrus.Debugf("Hosts: %v", ccrMo.Host)
```

Comment:
This and the log above look more like dev logs during testing. If you intend this to appear when customers run the installer with `--log-level debug` then I recommend wording them in a more friendly way.

Reply:
Good catch.  I'll update those.  Very useful for debugging what is coming back from vsphere.  Intention is more for support when issues occur to make sure we are able to see what vsphere is returning.